packages/slice-machine/src/components/Breadcrumb/Breadcrumb.css.ts

```diff
@@ -0,0 +1,5 @@
+import { style } from "@vanilla-extract/css";
+
+export const pageSpan = style({
+  fontWeight: "500",
```

Comment:
```suggestion
  fontWeight: 500,
```

We didn't use it as a string in our codebase with vanilla extarct yet